src/components/Collection/Details/Form.js

```diff
@@ -251,8 +251,8 @@ class CollectionForm extends Component {
             )}
           </FormItem>
 
-          <GroupLabel
-            label={<FormattedMessage id="mailingAddress" defaultMessage="Mailing address"/>}
+          <FormGroupHeader
```

Comment:
That's great! I was looking for that component but somehow missed it and invented my own bicycle.

Reply:
I did to - it is like yours, just different in styling and one for presentation and another for forms. Honestly I just assumed it wasn't in Antd